content/test/integration/service/deployments/deployment-pagination.spec.ts

```diff
@@ -0,0 +1,189 @@
+import { DeploymentResult, isSuccessfulDeployment, MetaverseContentService } from '@katalyst/content/service/Service'
+import { AuditInfo, EntityType, EntityVersion, SortingField, SortingOrder, Timestamp } from 'dcl-catalyst-commons'
+import { loadStandaloneTestEnvironment } from '../../E2ETestEnvironment'
+import { buildDeployData, EntityCombo } from '../../E2ETestUtils'
+/**
+ * This test verifies that when getting all deployments next link is paginating correctly
+ */
+describe('Integration - Deployment Pagination', () => {
+  let E1: EntityCombo, E2: EntityCombo, E3: EntityCombo
+
+  const testEnv = loadStandaloneTestEnvironment()
+  let service: MetaverseContentService
+
+  beforeAll(async () => {
+    const P1 = 'X1,Y1'
+    const type = EntityType.PROFILE
+
+    const timestamp = Date.now()
+    const a = await buildDeployData([P1], { type, timestamp, metadata: 'metadata1' })
+    const b = await buildDeployData([P1], { type, timestamp, metadata: 'metadata2' })
+    if (a.entity.id.toLowerCase() < b.entity.id.toLowerCase()) {
+      E1 = a
+      E2 = b
+    } else {
+      E1 = b
+      E2 = a
+    }
+    E3 = await buildDeployData([P1], { type, timestamp, metadata: 'metadata3' })
+  })
+
+  beforeEach(async () => {
+    service = await testEnv.buildService()
+  })
+
+  it('When there is no next page then the link is undefined', async () => {
+    // Deploy E1, E2 and E3 in that orderawait deploy(E1, E2, E3)
+
+    const actualDeployments = await service.getDeployments({
+      limit: 3
+    })
+
+    const nextLink = actualDeployments.pagination.next
+
+    expect(nextLink).toBeUndefined()
+  })
+
+  it('When lastEntityId is sent, it stills orders by timestamp', async () => {
+    // Deploy E1, E2 and E3 in that order
+    const [, E2Timestamp] = await deploy(E1, E2, E3)
+
+    const actualDeployments = await service.getDeployments({
+      limit: 2,
+      lastEntityId: E1.entity.id
```

Comment:
Does it make sense to test `lastEntityId` without a to/from timestamp filter? I mean, is it even used if none of those filters are present?

Also, let's make sure that we use if all 4 filters. from/to entity/local timestamp. And let's make sure that the returned deployments are actually the ones we expect.

Could make sense to make those tests in another test file, but we should do them